server/activityItem/queries.ts

```diff
@@ -217,16 +216,16 @@ export const createCollectionActivityItem = async (
 	actorId: null | string,
 	collectionId: string,
 ) => {
-	const collection: types.Collection = await Collection.findOne({ where: { id: collectionId } });
+	const collection = expect(await Collection.findOne({ where: { id: collectionId } }));
 	const { title } = collection;
 	return createActivityItem({
 		kind,
 		collectionId,
 		actorId,
-		communityId: collection.communityId,
+		communityId: expect(collection.communityId),
```

Comment:
These `expect`s were very tricky to do, because in some cases `null` is allowed to be passed and _is_ passed in the tests, causing the test to fail. 
This won't per se be a problem in prod because of how the `expect` function is designed to only throw in not-`production`, but still, failing tests are annoying.

In some cases (see `types/activity`) i elected to change the type of the ActivityItem itself to be e.g. `title | null`. I think there were only two cases where this was necessary though, at the very least for the review title.

Reply:
Another solution is to modify the tests here to e.g. correctly assign a title to the review, but I'll leave that up to you!